packages/firestore/test/integration/api/get_options.test.ts

```diff
@@ -566,16 +566,18 @@ apiDescribe('GetOptions', persistence => {
 
   it('get non existing doc while offline with source=server', () => {
     return withTestDocAndInitialData(persistence, null, docRef => {
-      return docRef.firestore
-        .disableNetwork()
-        // Attempt to get doc.  This will fail since there's nothing in cache.
-        .then(() => docRef.get({ source: 'server' }))
-        .then(
-          doc => {
-            expect.fail();
-          },
-          expected => {}
-        );
+      return (
+        docRef.firestore
```

Comment:
Mike, did you remove the parentheses manually? I know you hate them :)

Reply:
Bleh, that's so weird.  It seems like sometimes `git push` pushes the branch before running prettier, because my local branch has an "[AUTOMATED]: Prettier Code Styling" commit with this fix, but it apparently wasn't pushed. :-/